app/services/hearing_recorder.rb

```diff
@@ -8,13 +8,33 @@ def initialize(hearing_id:, body:, publish_to_queue:)
   end
 
   def call
-    hearing.update!(body: body)
-    HearingsCreatorWorker.perform_async(Current.request_id, hearing.id) if publish_to_queue
+    if hearing_contract.success?
+      hearing.update!(body: body)
+      publish_hearing_to_queue
+    else
+      Raven.capture_message("A hearing contract failed",
```

Comment:
Are you happy this will need changing for the sentry-ruby gem? Or is this worth changing now and merging the gem change first?